tests/model_selection/test_incremental.py

```diff
@@ -215,25 +240,30 @@ def test_search(c, s, a, b):
 
 
 @gen_cluster(client=True, timeout=None)
-def test_search_patience(c, s, a, b):
+def test_search_plateau_patience(c, s, a, b):
     X, y = make_classification(n_samples=100, n_features=5, chunks=(10, 5))
 
     class ConstantClassifier(SGDClassifier):
-        def score(*args, **kwargs):
-            return 0.5
 
-    model = ConstantClassifier(tol=1e-3)
+        def __init__(self, value=0):
```

Comment:
Do you need to set `self.value` here?

Reply:
Apparently not: I use `self.value` in `score`. But I don't understand how that works, so I added `self.value = value`.